sources/packages/backend/apps/api/src/app.module.ts

```diff
@@ -21,9 +21,18 @@ import { ConfigModule } from "@sims/utilities/config";
 import { DatabaseModule } from "@sims/sims-db";
 import { NotificationsModule } from "@sims/services/notifications";
 import { QueueModule } from "@sims/services/queue";
+import { HttpModule } from "@nestjs/axios";
 
 @Module({
   imports: [
+    {
+      ...HttpModule.register({
+        // `timeout` specifies the number of milliseconds before the request times out.
+        // If the request takes longer than `timeout`, the request will be aborted.
+        timeout: 5000,
```

Comment:
Shouldn't we have this coming from a config(an env property) ? I get that Idea that we are trying to keep it simple but I felt it must be something which could be modified at environment level when needed.

Reply:
As it was not defined in the ticket I refrain from mentioning it. If you are going ahead with this please keep in mind that the new ENV variable should be referenced on all PODs using the HTTP module.

Reply:
so keeping as it is

Reply:
We can create a small ticket for it(using env to be used in local/sources/devops)?

Reply:
I would not mind having it exposed as an ENV. Can be in a separated ticket.

Reply:
Added it as a system config constants